### PR TITLE
Adding relevant KeyManager changes to migration db scripts

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Purpose
Fixes https://github.com/wso2/product-apim/issues/8600

### CODE REVIEW

File: migration-scripts/migration-3.1.0_to_3.2.0/db2.sql [hunk 1]
```diff
@@ -45,3 +45,17 @@ CREATE TABLE IF NOT EXISTS AM_SHARED_SCOPE (
...
 ) /
 
+CREATE TABLE AM_KEY_MANAGER (
+  UUID VARCHAR(50) NOT NULL,
+  NAME VARCHAR(100) NULL,
+  DISPLAY_NAME VARCHAR(100) NULL,
+  DESCRIPTION VARCHAR(256) NULL,
+  TYPE VARCHAR(45) NULL,
+  CONFIGURATION BLOB NULL,
+  ENABLED INT NULL,
```
Comment (uvindra) [inline migration-scripts/migration-3.1.0_to_3.2.0/db2.sql:55]: Should be `ENABLED SMALLINT DEFAULT 0`. Please pick appropriate default value 0 or 1.

File: migration-scripts/migration-3.1.0_to_3.2.0/mssql.sql [hunk 2]
```diff
@@ -45,4 +45,18 @@ CREATE TABLE IF NOT EXISTS AM_SHARED_SCOPE (
...
+
+IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[AM_KEY_MANAGER]') AND TYPE IN (N'U'))
+CREATE TABLE AM_KEY_MANAGER (
+  UUID VARCHAR(50) NOT NULL,
+  NAME VARCHAR(100) NULL,
+  DISPLAY_NAME VARCHAR(100) NULL,
+  DESCRIPTION VARCHAR(256) NULL,
+  TYPE VARCHAR(45) NULL,
+  CONFIGURATION VARBINARY(MAX) NULL,
+  ENABLED BIT NULL,
```
Comment (uvindra) [inline migration-scripts/migration-3.1.0_to_3.2.0/mssql.sql:58]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: migration-scripts/migration-3.1.0_to_3.2.0/h2.sql [hunk 3]
```diff
@@ -5,7 +5,7 @@
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline migration-scripts/migration-3.1.0_to_3.2.0/h2.sql:8]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: migration-scripts/migration-3.1.0_to_3.2.0/mysql.sql [hunk 4]
```diff
@@ -5,7 +5,7 @@
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline migration-scripts/migration-3.1.0_to_3.2.0/mysql.sql:8]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: migration-scripts/migration-3.1.0_to_3.2.0/oracle.sql [hunk 5]
```diff
@@ -48,3 +48,17 @@ CREATE TABLE IF NOT EXISTS AM_SHARED_SCOPE (
...
 ) /
+
+CREATE TABLE AM_KEY_MANAGER (
+  UUID VARCHAR(50) NOT NULL,
+  NAME VARCHAR(100) NULL,
+  DISPLAY_NAME VARCHAR(100) NULL,
+  DESCRIPTION VARCHAR(256) NULL,
+  TYPE VARCHAR(45) NULL,
+  CONFIGURATION BLOB NULL,
+  ENABLED INT NULL,
```
Comment (uvindra) [inline migration-scripts/migration-3.1.0_to_3.2.0/oracle.sql:59]: This should be `ENABLED CHAR(1) DEFAULT 0`. Please pick appropriate default value 0 or 1.

File: migration-scripts/migration-3.1.0_to_3.2.0/postgresql.sql [hunk 6]
```diff
@@ -6,7 +6,7 @@ CREATE TABLE  IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BYTEA NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline migration-scripts/migration-3.1.0_to_3.2.0/postgresql.sql:9]: This should be `ENABLED BOOLEAN DEFAULT '1'`. Please pick appropriate default value 0 or 1.

File: migration-scripts/migration-3.1.0_to_3.2.0/db2.sql (hunk 1 above)
Comment (CrowleyRajapakse) [reply migration-scripts/migration-3.1.0_to_3.2.0/db2.sql]: resolved

File: migration-scripts/migration-3.1.0_to_3.2.0/mssql.sql (hunk 2 above)
Comment (CrowleyRajapakse) [reply migration-scripts/migration-3.1.0_to_3.2.0/mssql.sql]: resolved

File: migration-scripts/migration-3.1.0_to_3.2.0/h2.sql (hunk 3 above)
Comment (CrowleyRajapakse) [reply migration-scripts/migration-3.1.0_to_3.2.0/h2.sql]: resolved

File: migration-scripts/migration-3.1.0_to_3.2.0/mysql.sql (hunk 4 above)
Comment (CrowleyRajapakse) [reply migration-scripts/migration-3.1.0_to_3.2.0/mysql.sql]: resolved

File: migration-scripts/migration-3.1.0_to_3.2.0/oracle.sql (hunk 5 above)
Comment (CrowleyRajapakse) [reply migration-scripts/migration-3.1.0_to_3.2.0/oracle.sql]: resolved

File: migration-scripts/migration-3.1.0_to_3.2.0/postgresql.sql (hunk 6 above)
Comment (CrowleyRajapakse) [reply migration-scripts/migration-3.1.0_to_3.2.0/postgresql.sql]: resolved

**Pull request merged**